news_update.php

```diff
@@ -69,17 +69,15 @@
 
 echo '<div class="space-20"></div>';
 echo '<div class="alert alert-success">';
-echo lang_get( 'operation_successful' ) ?>
-
-echo '<br />';
+echo lang_get( 'operation_successful' );
+echo '<br /></div>';
 
 print_button( 'news_edit_page.php?news_id=' . $f_news_id . '&action=edit', lang_get( 'edit_link' ) );
+echo '&nbsp;&nbsp;&nbsp;&nbsp;';
```

Comment:
@badfiles I think you've summarized quite well. If you wanted to redistribute the code in your repository of mantisbt for instance, I think you'd then be unable to without purchasing a license yourself (of the admin template).

My understanding was that you couldn't include propriety code for this reason and was why a) a number of projects 'dual license' technology for scenarios like this. b) why people suggest that sometimes the MIT licenses should be used over the GPL.

From my point of view, it may be I need to contact the software freedom center with an initial email and see what advice they give. I spent over 10 years on the mantisbt project on that basis that the code would always be free to use for anyone in any way.